cluster/gce/config-default.sh

```diff
@@ -158,7 +158,7 @@ ENABLE_METRICS_SERVER="${KUBE_ENABLE_METRICS_SERVER:-true}"
 ENABLE_METADATA_AGENT="${KUBE_ENABLE_METADATA_AGENT:-none}"
 
 # Version tag of metadata agent
-METADATA_AGENT_VERSION="${KUBE_METADATA_AGENT_VERSION:-0.2-0.0.13-5}"
+METADATA_AGENT_VERSION="${KUBE_METADATA_AGENT_VERSION:-0.2-0.0.13-5-watch}"
```

Comment:
what's that? what's the versioning scheme of metadata agent?

Reply:
This version will change in a bit once the `watch` branch is merged into `master`. This scheme is: _<Dockerfile version>_-_<package version>_[-_<branch>_]...